tests/test_buffers.py

```diff
@@ -44,7 +44,7 @@ class DummyDictEnv(gym.Env):
     """
 
     def __init__(self):
-        self.action_space = spaces.Box(1, 5, (1,))
+        self.action_space = spaces.Box(1, 5, (10, 7))
```

Comment:
We might need to add a dedicated test for multi-dim action space.
As I'm a bit surprised it works for the rest of the algorithms (the line in the buffers was for the discrete actions initially), could you check that the output of the agent is correctly reshaped? (when using multiple env too)

It's in https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/common/policies.py#L333

I know it runs but I'm a bit afraid the order might be wrong due to the reshape (the agent only outputs 1D actions and then it gets reshape into multi-D array)